src/librustc/hir/mod.rs

```diff
@@ -2060,10 +2060,8 @@ pub struct InlineAsmOutput {
     pub span: Span,
 }
 
-// NOTE(eddyb) This is used within MIR as well, so unlike the rest of the HIR,
-// it needs to be `Clone` and use plain `Vec<T>` instead of `HirVec<T>`.
```

Comment:
Why did you lose this comment?

Reply:
That's probably a mistake; it was on `struct InlineAsm` originally and then I added `struct InlineAsmInner` but forgot to keep the comment on that one.

Reply:
r=me with this comment readded

Reply:
readded